scripts/kilocode/roomerge_cleanup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
I intended to do this in a seperate PR but accidentally committed it along (but I think that's fine)